manifests/m/Mikrotik/TheDude/7.15/Mikrotik.TheDude.locale.en-US.yaml

```diff
@@ -0,0 +1,20 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Mikrotik.TheDude
+PackageVersion: '7.15'
+PackageLocale: en-US
+Publisher: Mikrotik
+PackageName: The Dude
+PackageUrl: https://mikrotik.com/thedude
+License: Copyright (c) MikroTikls SIA

```

Comment:
A copyright isn't a license; Please update this with actual license terms such as Proprietary, MIT, GPL-2.0, or other terms as appropriate

Reply:
https://github.com/microsoft/winget-pkgs/blob/c9573c17e39efa2fbaa69b257561dc8857ce466e/manifests/m/Mikrotik/Winbox/3.40/Mikrotik.Winbox.locale.en-US.yaml#L12

TheDude as far as I know has the same license as Winbox.